wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights Tags and Categories: add drop-down rows for groups

Fixes #10832 

To test:

---
On a site that has a post with multiple tags/categories:
- Tip:  Hogwarts usually has some.
- Go to Stats > Insights > Tags and Categories.
- Verify:
  - The icon is `folderMultiple`.
  - The disclosure icon is pointed down.

<img width="350" alt="tandc_init" src="https://user-images.githubusercontent.com/1816888/52234527-1e8c9c80-287f-11e9-94d7-1b90035a17b2.png">

---
Click on a row that has multiple tags/categories.
- Verify:
  - Individual rows appear for each tag/category.
    - Max of 10 "child" rows will be displayed.
    - NOTE: we don't currently have the totals for expanded rows. Ref https://github.com/wordpress-mobile/WordPress-iOS/issues/10945.
  - On the "parent" row, the disclosure icon is pointed up.
  - The row dividers above and below the expanded section are the full width of the row.
  - Selecting a "child" row displays a webview for that tag/category.

<img width="350" alt="tandc_expanded" src="https://user-images.githubusercontent.com/1816888/52234578-3cf29800-287f-11e9-9621-e8857d9c92a0.png">

---
- Verify expanded rows persist on scrolling and rotation.
- Verify expanded rows are reset when Insights is first displayed.
